templates/policies/sts_installer_permission_policy.json

```diff
@@ -111,11 +111,18 @@
                 "elasticloadbalancing:SetLoadBalancerPoliciesOfListener",
                 "iam:AddRoleToInstanceProfile",
                 "iam:CreateInstanceProfile",
+                "iam:CreateRole",
```

Comment:
Is this actually part of the 4.10 installer permissions now? I think customers might be concerned about this change.

cc @arendej

Reply:
@vkareh @arendej  it does based on https://github.com/openshift/openshift-docs/blob/enterprise-4.10/modules/installation-aws-permissions.adoc

Reply:
Are we certain it's necessary to add this? we really shouldn't be adding iam:CreateRole. Is it possible it's there for OCP but we don't use it?

Reply:
I doubt this is a new required permission. OCP shouldn't be creating any role that's not predefined in CredentialRequests, we today pre create all roles defined by CredentialRequests extracted from the release.

Reply:
How can we determine what's used and what's not? Not sure I want to blindly remove permissions that look wrong without knowing what it's used for. On the other hand, why would the OCP/Docs team add it if it's not necessary?

Reply:
The permission existed in 4.9 https://github.com/openshift/openshift-docs/blob/enterprise-4.9/modules/installation-aws-permissions.adoc

We Managed Service remove it because its not required for STS, we're not blindly removing and we can only validate it through testing until we get a supported STS only policy from the installer.